### DISCUSSION TITLE
Limit command usage to plot

### DISCUSSION THREAD

**Discussion by confusedalex** [TownyAdvanced/Towny#5909] [Q&A] [answered]
Hey, I have a economy plugin on my server and want to limit the usage of /deposit and /withdraw to bank plots. Is there any way I could do this?

**Comment by LlmDl** [comment, marked as answer]
```
bank:
 
  # If true players will only be able to use /t deposit, /t withdraw, /n deposit & /n withdraw while inside bank plots belonging to the town or nation capital respectively.
  # Home plots will also allow deposit and withdraw commands.
  is_banking_limited_to_bank_plots: 'true'
  ```

**Reply by confusedalex** [reply]
Thanks for your fast answer but I've already set this option to true. I wanted to limit the withdraw and deposit commands for the players account. Is there any way to limit a command usage to a plot type. Or that players only got a command permission when they are on a bank plot?

**Reply by LlmDl** [reply]
What plugin supplies those withdraw and deposit commands?

**Reply by confusedalex** [reply]
It's "[The Gold Bank](https://www.spigotmc.org/resources/the-gold-bank.70562/)". I know that this specific plugin doesn't have a setting for this in the config file. I thought it could be possible to limit this via Vault or the permissions.

**Reply by LlmDl** [reply]
Ideally, the gold bank would add support for Towny's bank plots. It has not been updated in a while though and I do not see any source code offered.

**Reply by confusedalex** [reply]
So this would be something the plugin author would have to implement?

**Reply by LlmDl** [reply]
That, or if TheGoldBank has an API with events for deposit/withdraw, I could make you a bridge plugin for a fee.

**Reply by confusedalex** [reply]
Unfortunately TheGoldBank doesn't have an API and the Plugin seems dead to me. I really appreciate your fast and good help!

**Reply by LlmDl** [reply]
You might consider using TheNewEconomy paired with my plugin Towny-TNE.
